recipes/abseil/all/conanfile.py

```diff
@@ -30,35 +30,54 @@ class AbseilConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-
     short_paths = True
 
+    @property
+    def _min_cppstd(self):
+        return 11 if Version(self.version) < "20230125.0" else 14
+
+    @property
+    def _compilers_minimum_version(self):
+        if self._min_cppstd == 11:
+            return {}
+        return {
+            "gcc": "6",
+            "clang": "5",
+            "apple-clang": "10",
+            "Visual Studio": "15",
+            "msvc": "191",
+        }
```

Comment:
```suggestion
        return "11" if Version(self.version) < "20230125.0" else "14"

    @property
    def _compilers_minimum_version(self):
        return {
            "14": {
                "gcc": "6",
                "clang": "5",
                "apple-clang": "10",
                "Visual Studio": "15",
                "msvc": "191",
            },
        }.get(self._min_cppstd, {})
```

I think it's more elegant.

Reply:
What is the policy for setting these minimum compiler versions? They must be provided by the project itself (folly) or is something that can be freely set by CCI? Wouln't it be good to set minimum defaults for C++11 (or use `conan.tools.build.supported_cppstd`)?

Reply:
Projects usually don't know the full range of supported compilers. They test few versions. Moreover for a given compiler, the range between "0 support of C++xx" and " full support of C++××" can be huge. Many projects using common C++11 features can be built by gcc4.9 while it has not full C++11 support.